mlir/include/mlir/Dialect/SCF/IR/DeviceMappingInterface.td

```diff
@@ -60,8 +60,51 @@ def DeviceMappingAttrInterface : AttrInterface<"DeviceMappingAttrInterface"> {
   ];
 }
 
+def DeviceMaskingAttrInterface : AttrInterface<"DeviceMaskingAttrInterface"> {
+  let cppNamespace = "::mlir";
+  let description = [{
+    Attribute interface describing how to filter the processing units that a
+    region is mapped to.
+
+    A popcount can be applied to determine the logical linear index that a
+    physical processing unit is responsible for.
```

Comment:
I'm not sure this part of the documentation is understandable by itself. It seems to imply that the attribute is systematically a bitfield, and I'm not sure I fully understand how a popcount is always needed to get a logical linear index.